UnityProject/Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -351,11 +351,11 @@ internal set
                 }
 
                 // old not empty
-                if (!string.IsNullOrEmpty(oldAssetIdSrting))
-                {
-                    Debug.LogError($"Can not Set AssetId on NetworkIdentity '{name}' because it already had an assetId, current assetId '{oldAssetIdSrting}', attempted new assetId '{newAssetIdString}'");
-                    return;
-                }
+                // if (!string.IsNullOrEmpty(oldAssetIdSrting))
+                // {
+                    // Debug.LogError($"Can not Set AssetId on NetworkIdentity '{name}' because it already had an assetId, current assetId '{oldAssetIdSrting}', attempted new assetId '{newAssetIdString}'");
+                    // return;
+                // }
```

Comment:
don't comment code, just remove it. We can retrieve it with git if we need to.

Reply:
ok